datasets/acute-care/extraction_wrapper.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+"""
+Wrapper script to initiate a hail batch, and submit using an analysis-runner managed dataproc cluster
+"""
+import hailtop.batch as hb
+import os
+import sys
+
+from analysis_runner import dataproc
+
+
+def main():
+    """
+    Create a Hail Batch
+    Use the analysis-runner helper to generate a DataProc cluster, and add the job
+    Set off the batch
+    """
+
+    service_backend = hb.ServiceBackend(
+        billing_project=os.getenv('HAIL_BILLING_PROJECT'), bucket=os.getenv('HAIL_BUCKET')
+    )
+
+    # create a hail batch
+    batch = hb.Batch(name='cohort_mt_extraction', backend=service_backend)
+
+    my_cluster = dataproc.hail_dataproc_job(
+        batch=batch,
+        script=" ".join(sys.argv[1:]),
+        max_age='4h',
+        job_name="extract_from_cohort_mt",
```

Comment:
```suggestion
        job_name='extract_from_cohort_mt',
```

Reply:
Just a stylistic question, but the default linting I use is Black, which prefers ". Would you prefer use of '?

Reply:
@MattWellie, we decided that we would use single quotes to be consistent with Hail codebase (see https://github.com/populationgenomics/team-docs/blob/main/new_repository.md#linters). 
You might want to copy linter configs from https://github.com/populationgenomics/team-docs/tree/main/new_repository (black's config in `pyproject.toml` uses `skip-string-normalization = true` to make it ignore quotes). 

You may also want to run `pre-commit install --install-hooks` in the repo to enable pre-commit if you haven't already!